Code.of.Conduct.DRiP.md

```diff
@@ -0,0 +1,61 @@
+---
+tags: 🌱/٤/🟩
+type: 🌱
+subject: "DRiP"
+subsubject: "Code-of-Conduct"
+status: 🟩
+last-reviewed: 2023-08-26
+creation-date: 2023-08-26
+author: Christophe Pere
+---
+
+# Code of Conduct DRiP
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors, maintainers, and leaders pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behaviour that contributes to creating a positive environment include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy toward other community members
+
+Examples of unacceptable behaviour include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or advances
+- Trolling, insulting/derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or electronic address, without explicit permission
+- Other conduct that could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities
+
+Project maintainers and community leaders are responsible for clarifying the standards of acceptable behaviour. They are expected to take appropriate and fair corrective action in response to any instances of unacceptable behaviour.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct or to ban temporarily or permanently any contributor for behaviours that they deem inappropriate, threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within and in public spaces when an individual represents the project or its community. Examples of representing a project or community include using an official project email address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behaviour may be reported by contacting the project team at [[your-email@example.com](mailto:your-email@example.com)] (or create a form as qiskit COC). All complaints will be reviewed and investigated, resulting in a necessary and appropriate response to the circumstances. The project team is obligated to maintain confidentiality concerning the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
I think for now we can use my email address and say something like "...reported by contacting the Aggie Branczyk at [[aggiebranczyk@gmail.com](mailto:aggiebranczyk@gmail.com)]". Later we can think about what is the best way to set up communication with a broader team. 

Also, can we modify the part that says "or create a form as qiskit COC" to reflect DRiP?